airbyte-integrations/connectors/destination-mysql-strict-encrypt/src/test-integration/java/io/airbyte/integrations/destination/mysql/MySqlTestDataComparator.java

```diff
@@ -30,7 +33,35 @@ protected boolean compareBooleanValues(String firstBooleanValue, String secondBo
     if (secondBooleanValue.equalsIgnoreCase("true") || secondBooleanValue.equalsIgnoreCase("false")) {
       return super.compareBooleanValues(firstBooleanValue, secondBooleanValue);
     } else {
-      return super.compareBooleanValues(firstBooleanValue, String.valueOf(secondBooleanValue.equals("1")));
+      return super.compareBooleanValues(firstBooleanValue,
+              String.valueOf(secondBooleanValue.equals("1")));
+    }
+  }
+
+  @Override
+  protected boolean compareDateTimeValues(String expectedValue, String actualValue) {
```

Comment:
How is this work relevant to the error messages work?

Reply:
this logic is added here in the strict-encrypt connector from the main connector. Here is the number of the pull request and its name in which it was done - `[15245] Destination-mysql: fixed normalization tests after changes in python part`. The developer who added these changes only added them to the main connector, but forgot about strict-encrypt. Therefore, the tests will fail here, which interferes with testing and publishing.